### ISSUE TITLE
Have a hidden NPC list for the NPC Indicator Plugin

### ISSUE THREAD

**Issue by tikeyike** [runelite/runelite#7147]
**Is your feature request related to a problem? Please describe.**
When questing I like using the NPC Indicator plugin to have the names of the NPCs appear above their heads, so I use the "*" to highlight all the NPCs instead of typing in each NPC that I want highlighted. But it is kind of annoying that EVERYTHING is highlighted; things like butterflies, bankers, rats and so on.

**Describe the solution you'd like**
If we could have an NPC Hidden list, similar to the one used in the Ground Items plugin, where you can put items on a list that you don't want to show up. This would be nice if it could be don't for NPCs to filter out the NPCs that you don't want highlighted.

**Describe alternatives you've considered**
An alternative is to just type in the names of the NPCs that you want highlighted, but I'd rather have all NPCs highlighted and then filter to hide NPCs that I don't want highlighted.

**Comment by tikeyike** [comment]
Yeah I guess it comes across that way. I thought it would be nicer if you can filter NPCs like how to it with ground items. I think its annoying to type in like 3 to 6 unique NPC names for every quest if I want to highlight the NPCs involved in the quest to see them more easily. In the end it's just a suggestion, I guess people don't use it the same way as me.

**Comment by deathbeam** [comment]
I don't mind this I just think this use-case is odd. Added appropriate labels anyway
